src/components/AdjustLeveragePosition/AdjustLeveragePosition.tsx

```diff
@@ -193,9 +252,7 @@ const AdjustLeveragePosition: FC<AdjustPositionProps> = ({
       return null;
     }
 
-    return principalCollateralBalance.add(
-      isAddCollateral ? underlyingCollateralAmount : underlyingCollateralAmount.mul(-1),
-    );
+    return principalCollateralBalance.add(underlyingCollateralAmount).mul(isAddCollateral ? 1 : -1);
```

Comment:
I think this is not good in case we are withdrawing. In case user current principal balance is 5 and user is withdrawing 3. Previously we would get 2 as a result. With new formula we will get -8.

Reply:
updated